README.md

```diff
@@ -1,10 +1,79 @@
+[![Build Status](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_apis/build/status/Sitecore.jss?branchName=master)](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_build/latest?definitionId=1?branchName=master) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md)
 
 This repository contains source code for all Sitecore JSS packages and templates to help you get started using Sitecore JSS.
 
 ## Getting started with JSS
+To develop a JSS application, you need:
+- An operating system supported by Node (Mac, Windows, Linux).
+- Node. We recommend using the latest long-term support (LTS) release.
+
+> To run a JSS application in production or develop using Sitecore data you need to connect your application to a Sitecore instance with the Headless Services module installed (requires a **Sitecore Headless Services license**). Sitecore *requires Windows*, but the Sitecore instance does not need to be local. You can use a Sitecore instance installed in a virtual machine or on a remote server.
+
+### Identify the JSS version for your Sitecore version
+JSS versions are coupled with Sitecore versions. The current JSS release is compatible with the latest version of Sitecore. 
+
+If you use an older version of Sitecore, before creating a JSS project, you need to determine which JSS version is compatible with your version of Sitecore XP.
+
+To identify the correct JSS version for your Sitecore XP version:  
+- For Sitecore XP 7.5—9.3, in the [Sitecore modules compatibility table for Sitecore XP 7.5—9.3](https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB0541788), in the section *Headless Services*, identify the correct JavaScript Services version for your Sitecore version.
+- For Sitecore XP 10.0 and later, in the [Sitecore modules compatibility table for Sitecore XP 10.0 and later versions](https://support.sitecore.com/kb?id=kb_article_view&sysparm_article=KB1000576), in the section *Headless Services*, identify the correct JavaScript Services version for your Sitecore version.
+
+### Getting started with the latest version of JSS
+  <!---
+   @TODO: Update to version 20.0.0 docs before release
+   -->
+To create a JSS project, perform the following steps:
+1. Install the JSS CLI:   
+	```
+	npm install -g @sitecore-jss/sitecore-jss-cli
+	```
+2. In a terminal, run the following command to create the JSS application:
+	```
+	jss create <your-app-name> <app-template-name>
+	```	
+	
+	Replace `<your-app-name>` with your desired application name, and `<app-template-name>` with the name of the template. 
+	
+	For example: `jss create my-first-jss-app nextjs`.
+	
+	>This command scaffolds a JSS application based on the ***latest version of JSS templates*** regardless of the JSS CLI version, because it pulls the template code from the `master` branch of the JSS repository.
```

Comment:
Steps 1&2 are combined now. e.g.:
```
npm init sitecore-jss nextjs
```
or
```
npx create-sitecore-jss nextjs
```

Can probably get rid of the disclaimer (about branch usage)? Since templates are packaged in `create-sitecore-jss`, it's just a matter of using the proper version tag of the npm package.